src/readline.cpp

```diff
@@ -0,0 +1,28 @@
+#include <Rcpp.h>
+using namespace Rcpp;
+
+#include <Rinterface.h>
+
+extern "C" int R_ReadConsole(const char*, unsigned char*, int, int);
```

Comment:
If you do have trouble with this (e.g. on Windows) then you might also be able to import `ptr_R_ReadConsole` (although we would want to see if R_ReadConsole does any wrapping of that call we'd want to emulate)